### ISSUE TITLE
Make proxy_uri_header configurable

### ISSUE THREAD

**Issue by micw** [bitnami/bitnami-docker-matomo#27]
**Description**

To use matomo behind a proxy and map it to a sub path, it is required to set proxy_uri_header = 1 in global.ini.php. Please allow to pass this via environment variable.

https://matomo.org/faq/how-to-install/faq_98/

**Comment by javsalgar** [comment]
Thanks @dherges, so you tested it and it worked without issues, right?

**Comment by dherges** [comment]
@javsalgar No, I couldn't get it working in the `bitnami/matomo` image, but it works in the root `matomo` image: https://github.com/matomo-org/docker/blob/master/.examples/apache/docker-compose.yml#L10

**Comment by micw** [comment]
IMO it does not make sense to provide a complete config just to set a single option. That would mean that on each update the individual config would need to be checked/updated.

**Comment by javsalgar** [comment]
As soon as this feature is available in the container, I will update this thread.

**Comment by overclockedllama** [comment]
Any updates? I want to run matomo behind nginx

**Comment by juan131** [comment]
Hi @overclockedllama @micw 

We just released a new revision of the Matomo image (**3.11.0-r22**) that includes a new environment variable **MATOMO_ENABLE_PROXY_URI_HEADER** that allows you to configure the `proxy_uri_header` option in the **config.ini.php** configuration file. Find more information in the [README.md](https://github.com/bitnami/bitnami-docker-matomo#configuration)

You can use the **docker-compose.yaml** below to test this new environment variable:

```Dockerfile
version: '2'
services:
  mariadb:
    image: 'bitnami/mariadb:10.3'
    environment:
      - MARIADB_USER=bn_matomo
      - MARIADB_DATABASE=bitnami_matomo
      - ALLOW_EMPTY_PASSWORD=yes
    volumes:
      - 'mariadb_data:/bitnami'
  matomo:
    image: 'bitnami/matomo:3'
    environment:
      - MARIADB_HOST=mariadb
      - MARIADB_PORT_NUMBER=3306
      - MATOMO_DATABASE_USER=bn_matomo
      - MATOMO_DATABASE_NAME=bitnami_matomo
      - MATOMO_ENABLE_PROXY_URI_HEADER=yes 
      - ALLOW_EMPTY_PASSWORD=yes
    ports:
      - '80:80'
      - '443:443'
    depends_on:
      - mariadb
    volumes:
      - 'matomo_data:/bitnami'
volumes:
  mariadb_data:
    driver: local
  matomo_data:
    driver: local
```

```bash
$ docker-compose up -d
## Wait for Matomo to be initialized...
$ docker-compose exec matomo cat /opt/bitnami/matomo/config/config.ini.php | grep 'proxy_uri_header'
proxy_uri_header=1
```

**Comment by carrodher** [comment]
Please, feel free to reopen the issue if the issue persists
